rct229/schema/ASHRAE229.schema.json

```diff
@@ -559,11 +559,11 @@
                 },
                 "status_type": {
                     "description": "Choice of new, existing, addition, alteration, etc. for each ruleset.",
-                    "oneOf": [
+                    "anyOf": [
                         {
                             "$ref": "Enumerations2019ASHRAE901.schema.json#/definitions/SpaceStatusOptions2019ASHRAE901"
                         },
-                        {
+                                                {
```

Comment:
These spaces also don't need to be added.

Reply:
Good catch!